### PR TITLE
feat: get the SDK to run without any `yarn build` step

### PR DESCRIPTION
**Pull request by michaelfig**
Note that the vat-admin vat needs rework to use bundles instead of
evaluable strings.  For now, those tests are skipped.

Also, take a step towards transforms that don't get their hands on `endowments` by having the `default-evaluate-options` supply a shim for HandledPromise instead of it being part of the `transform-eventual-send` transform.

### CODE REVIEW

File: packages/SwingSet/src/controller.js [hunk 1]
```diff
@@ -296,7 +285,8 @@ export async function buildVatController(config, withSES = true, argv = []) {
     vatAdminVatSetup: await evaluateToSetup(ADMIN_VAT_PATH, 'vat-vatAdmin'),
   };
 
-  const kernel = buildSESKernel(sesEvaluator, kernelEndowments);
+  const kernelSetup = await evaluateToSetup(KERNEL_PATH, 'kernel');
```
Comment (warner) [inline packages/SwingSet/src/controller.js:288]: `evaluateToSetup` is now misnamed, if we're also using it to load the kernel (which doesn't provide a `setup` function)

File: packages/SwingSet/test/test-vat-imports.js [hunk 2]
```diff
@@ -9,8 +9,13 @@ async function requireHarden(t, withSES) {
 }
 
 test('vat can require harden with SES', async t => {
-  await requireHarden(t, true);
-  t.end();
+  try {
+    await requireHarden(t, true);
+  } catch (e) {
+    t.isNot(e, e, 'unexpected exception');
```
Comment (warner) [inline packages/SwingSet/test/test-vat-imports.js:15]: why not use `t.fail`?

File: packages/SwingSet/test/vat-admin/test-innerVat.js [hunk 3]
```diff
@@ -25,7 +23,7 @@ async function testVatCreationFromBuild(t, withSES) {
   t.end();
 }
 
-test('VatAdmin inner vat creation', async t => {
+test.skip('VatAdmin inner vat creation', async t => {
```
Comment (warner) [inline packages/SwingSet/test/vat-admin/test-innerVat.js:26]: can't really land this until all the previous tests are passing, we don't want to let vat-admin regress

File: packages/agoric-cli/package.json [hunk 4]
```diff
@@ -11,9 +11,8 @@
   "scripts": {
     "build": "exit 0",
     "test": "tape -r esm 'test/**/*.js'",
-    "lint": "eslint '**/*.{js,jsx}'",
+    "lint-check": "eslint '**/*.{js,jsx}'",
```
Comment (warner) [inline packages/agoric-cli/package.json:14]: please put all these lint improvements (which are awesome) in a separate PR

File: packages/SwingSet/test/vat-admin/test-innerVat.js (hunk 3 above)
Comment (michaelfig) [reply packages/SwingSet/test/vat-admin/test-innerVat.js:26]: Agreed.  I'll wait for your new SES to land before I try to work on this PR further.

File: packages/SwingSet/test/test-vat-imports.js (hunk 2 above)
Comment (michaelfig) [reply packages/SwingSet/test/test-vat-imports.js:15]: `t.fail(e)` does not print the exception stack trace.

File: packages/SwingSet/src/controller.js (hunk 1 above)
Comment (michaelfig) [reply packages/SwingSet/src/controller.js:288]: Agreed.

**Pull request closed without merging**